Rubberduck.SourceControl/GitProvider.cs

```diff
@@ -246,7 +247,15 @@ public override void Fetch([Optional] string remoteName)
 
                 if (remote != null)
                 {
-                    _repo.Network.Fetch(remote);
+                    //_repo.Network.Fetch(remote); // deprecated on LibGit2Sharp from v0.24.0
+                    /* 
+                     * The new functionality requires a refSpec. 
+                     * As I suppose we're just tracking by default the whole remote,
+                     * then I choose to hardcode the refSpec here:
+                     */
+                     // NOTE: hardcoded string
```

Comment:
the multi-line comment is meta-commentary. It only justifies the change as a diff, but doesn't make sense when the diff isn't there. Remove it.  
Convert the "NOTE" to a "FIXME" and consider opening a technical-debt issue about it.

also: outdated code can be deleted :)

Reply:
Sure, sorry for the excesses in commenting :)